src/Makefile.am

```diff
@@ -1,24 +1,30 @@
 #
-# Copyright (C) Mellanox Technologies Ltd. 2019.  ALL RIGHTS RESERVED.
+# Copyright (C) Mellanox Technologies Ltd. 2019-2020.  ALL RIGHTS RESERVED.
 # See file LICENSE for terms.
 #
 
 lib_LTLIBRARIES     = libnccl-net.la
 
 libnccl_net_la_CFLAGS   = $(CFLAGS) -I$(top_srcdir)/include -DGNU_SOURCE
-libnccl_net_la_LIBADD   = -lsharp_coll -lcudart_static
+libnccl_net_la_LIBADD   = -lcudart_static
```

Comment:
was it cudart_static always? did we change it recently?

Reply:
I didn't change this, I think it always was static. If you think about memtype cache issue then it's probably not so important because we don't have any cudamalloc in plugins and moreover cudart_static is default in nccl.